.github/workflows/ruby_ci.yml

```diff
@@ -0,0 +1,54 @@
+
+name: Ruby Lint and Test
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    env:
+      TZ: Europe/London
+      DISPLAY: ':99.0'
+    # Services
+    # https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservices
+    services:
+      mysql:
+        # Use the Mysql docker image https://hub.docker.com/_/mysql
+        image: mysql:5.7 # Using 5.7 to map to what is used in production.
+        ports:
+         - 3306:3306 # Default port mappings
+         # Monitor the health of the container to mesaure when it is ready
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+        env:
+          MYSQL_ROOT_PASSWORD: '' # Set root PW to nothing
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # Runs bundle install and caches gems. See the ruby_test.yml
+                            # example if you need more control over bundler.
+    - name: Setup database
+      run: bundle exec rake db:setup
+    - name: Test & publish code coverage
+      uses: paambaati/codeclimate-action@v2.7.5
+      env:
+        CC_TEST_REPORTER_ID:  ${{ secrets.CC_TEST_REPORTER_ID || '5e399530a457db7a41cd5785ce0536b79e9022b3c2d4382f101310b3b166eb38' }}
+      with:
+        coverageCommand: bundle exec rspec
+  lint:
```

Comment:
Does it matter to test before linting?

Reply:
They are separate jobs, so happen in parallel.